README.md

```diff
@@ -1 +1,17 @@
-# codacy-test
\ No newline at end of file
+# codacy-test
+
+### Among Codacy’s features
+- Identify new Static Analysis issues
```

Comment:
Codacy has a fix for the issue: [Expected: dash; Actual: asterisk](https://app.codacy.com/gh/franciscodua/codacy-test/pullRequest?prid=7573067)
```suggestion
- Commit and Pull Request Analysis with GitHub, BitBucket/Stash, GitLab (and also direct git repositories)
```